src/query/shorthand.ts

```diff
@@ -53,7 +56,13 @@ export function spec(specQ: SpecQuery,
     parts.push(value(specQ.mark, replace[Property.MARK]));
   }
 
-  // TODO: transform
+  if (specQ.transform) {
+    const calc = stringifyCalculate(specQ.transform.calculate);
+    const filter = stringifyFilter(specQ.transform.filter);
+
+    parts.push((calc ? `calculate:${calc}` : '') +
```

Comment:
@kanitw I'm not sure what the best way to divide calculate and and filter would be, especially since there are occasions where one or the other may not appear. Should we leave it as is, or do you have a specific delimiter in mind?